src/markdown-pages/automate-workflows/automated-tagging.mdx

```diff
@@ -0,0 +1,312 @@
+[✔] Installing On-Host instrumentation success
+[✔] Installing Services and instrumentations success
+[INFO] Deployment successful!
+
+Deployed Resources:
+
+  host1 (aws/ec2):
+    ip: 52.90.86.109
+    services: ["app1"]
+    instrumentation:
+       nr_infra: newrelic v1.12.1
+
+
+Installed Services:
+
+  app1:
+    url: http://52.90.86.109:5001
+    instrumentation:
+       nr_node_agent: newrelic v6.11.0
+
+Completed at 2020-08-20 15:11:14 +0000
+```
+
+</Step>
+
+<Step>
+
+## Generate Traffic
+The simple node application that was deployed contains a very simple web page that looks similar to the screen shot below. Follow the instructions on this page to generate traffic that will be sent to New Relic.
+![Hello World Web Site](../../images/automate-helloworld/helloworld-automationdemo.png)
+
+</Step>
+
+<Step>
+
+## Check out these tags!
+Let's head on over to New Relic. We'll use the [search redirect](https://one.newrelic.com/redirect/search/Hello) to quickly find the resources we deployed. (Folks in our European data center should use this [link](https://one.eu.newrelic.com/redirect/search/Hello).)
+
+![Search Redirect](../../images/automate-helloworld/global-search.png)
+
+</Step>
+
+<Step>
+Once you are in New Relic, you can select either the host or the app that was deployed. By clicking on the 'i' icon next to the name, you'll open the metadata and tags modal. Notice that all the tags in the "global_tags" section are applied to both entities. 
+
+![Search Redirect](../../images/automate-helloworld/host-tags.png)
+
+![Search Redirect](../../images/automate-helloworld/app-tags.png)
+
+</Step>
+
+<Step> 
+## Extra Credit: Create your own configuration file
+
+Create your own configuration and change the global tags! One cool thing you can do is simply pass in a configuration file URL to the deployer. Here's one that uses a gist on GitHub:
+
+```bash
+docker run -it\
+    -v $HOME/configs/:/mnt/deployer/configs/\
+    --entrypoint ruby deployer main.rb -c configs/[user].docker.local.json -d https://gist.githubusercontent.com/markweitzel/d281fde8ca572ced6346dc25470790a5/raw/a227a802aa7a63819e688a383b575e7fd28223ee/MarksHelloDemoConfig.json
+```
+
+
+</Step>
+
+<Step>
+
+## Teardown
+Because we've provisioned resources in your cloud account, you'll want to ensure to decommision them. This can also be done with the deployer. Add the 'teardown' flag, -t to the end of your command. 
+
+```bash
+docker run -it\
+    -v $HOME/configs/:/mnt/deployer/configs/\
+    --entrypoint ruby deployer main.rb -c configs/[user].docker.local.json -d documentation/tutorial/user_stories/Hello/hello.json -t
+```
+
+</Step>
+
+
+
+</Steps>
+
+
+<br />
+
+---
+We hope you enjoyed this guide. As you can tell, this is very much a work in progress, but we are excited about getting it in your hands and would love your feedback! Hit us up in the [Build on New Relic](https://discuss.newrelic.com/c/build-on-new-relic) and let us know what you think! 
```

Comment:
```suggestion
We hope you enjoyed this guide. As you can tell, this is very much a work in progress, but we are excited about getting it in your hands and would love your feedback! Hit us up in the [Build on New Relic](https://discuss.newrelic.com/c/build-on-new-relic) support thread and let us know what you think! 
```

Feel free to pick different verbage if you don't like this suggestion. This felt like it needed something to qualify what "Build on New Relic" was. It reads a bit funny without specifying this as the support or discussion thread.